workers/unity/Assets/Playground/Scripts/Worker/IosClientWorkerConnector.cs

```diff
@@ -0,0 +1,59 @@
+using Improbable.Gdk.Core;
+using Improbable.Gdk.Mobile;
+#if UNITY_IOS
+using Improbable.Gdk.Mobile.Ios;
+#endif
+using System;
+using UnityEngine;
+
+namespace Playground
+{
+    public class iOSClientWorkerConnector : MobileWorkerConnector, IMobileConnectionController
+    {
+        public string IpAddress { get; set; }
+        public ConnectionScreenController ConnectionScreenController { get; set; }
+
+        [SerializeField] private GameObject level;
+
+        private GameObject levelInstance;
```

Comment:
Does this not need to be removed on dispose or destroy? (Or disable)

Reply:
yep, seems like that go removed by accident. will add it back in

Reply:
added it in.